compiler/src/main/java/ca/mcscert/jpipe/model/elements/JustificationModel.java

```diff
@@ -1,9 +1,11 @@
 package ca.mcscert.jpipe.model.elements;
 
+import ca.mcscert.jpipe.model.RepTable;
 import ca.mcscert.jpipe.model.SymbolTable;
 import ca.mcscert.jpipe.model.Visitable;
 import ca.mcscert.jpipe.model.cloning.Replicable;
 import java.util.Collection;
+import java.util.HashMap;
```

Comment:
maybe should just be a Map. Anyway "will change in the future" (famous last words)